### DISCUSSION TITLE
Geburtstag in den Kalender mit Erinnerung

### DISCUSSION THREAD

**Discussion by Taste-IT** [tine20/tine20#7373] [Q&A] [answered]
Hi,
ich habe meine Kontakte mit Geburtstag hinterlegt. Diese werden aber per se nicht in den Kalender eingetragen. Was muss ich machen, damit die Geburtstage auch im Kalender und idealerweise auch mit Reminder hinterlegt werden?
thx

**Comment by Taste-IT** [comment]
Hi,
keiner eine Idee?

**Comment by pschuele** [comment, marked as answer]
hi,
ist leider aktuell nicht möglich. hier ist der dazugehörige feature request: https://github.com/tine20/tine20/issues/784
